File: python/ejemplo_03.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import gi
try:
    gi.require_version('Gtk', '3.0')
except ValueError:
    exit(1)
from gi.repository import Gtk


if __name__ == '__main__':
    dialog = Gtk.MessageDialog()
    dialog.set_title('https://www.atareao.es')
    dialog.set_markup('¿Estás <span weight="bold" foreground="red">seguro</span>?')
    dialog.add_button('Si', Gtk.ResponseType.YES)
    dialog.add_button('No', Gtk.ResponseType.NO)
    try:
        if dialog.run() == Gtk.ResponseType.YES:
            exit(0)
    except KeyboardInterrupt:
        pass
    exit(1)

File: javascript/ejemplo_03.js
#!/usr/bin/gjs

imports.gi.versions.Gtk = '3.0';
const {Gtk} = imports.gi;

Gtk.init(null);

function show_dialog(){
    let dialog = new Gtk.MessageDialog({
        title: 'https://www.atareao.es',
        text: '¿Estás <span weight="bold" foreground="red">seguro</span>?',
        use_markup: true
        });
    dialog.add_button('Si', Gtk.ResponseType.YES);
    dialog.add_button('No', Gtk.ResponseType.NO);
    try{
        if(dialog.run() == Gtk.ResponseType.YES){
            return 0;
        }
    }catch(error){
    }
    return 1;
}
show_dialog();
